mobile/native/src/api.rs

```diff
@@ -30,3 +33,24 @@ pub fn calculate_liquidation_price(
         price, leverage, direction,
     ))
 }
+
+pub fn run(stream: StreamSink<Event>, app_dir: String) -> Result<()> {
+    anyhow::ensure!(!app_dir.is_empty(), "app_dir must not be empty");
```

Comment:
:question: What does this condition mean? The message is unclear to me.

Reply:
I removed the assertion - as not really needed.